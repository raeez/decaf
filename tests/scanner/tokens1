// Decaf keywords
boolean
callout
class
else
false
if
int
return
true
void
for
forpar
break
continue
