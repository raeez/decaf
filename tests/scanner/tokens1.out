2 RIDENTIFIER boolean
3 RIDENTIFIER callout
4 RIDENTIFIER class
5 RIDENTIFIER else
6 BOOLEANLITERAL False
7 RIDENTIFIER if
8 RIDENTIFIER int
9 RIDENTIFIER return
10 BOOLEANLITERAL True
11 RIDENTIFIER void
12 RIDENTIFIER for
13 IDENTIFIER forpar
14 RIDENTIFIER break
15 RIDENTIFIER continue
16 EOF
